### ISSUE TITLE
Miniwindows Snap to Grid

### ISSUE THREAD

**Issue by AlisonMAir** [fiendish/aardwolfclientpackage#318]
A few people on my ftalk have been talking about this/asking for this lately, so... 

Is it possible to make miniwindows "snap to grid" or otherwise automatically align with the main window's edge?

**Comment by fiendish** [comment]
Yes, but please tell me more about how you imagine it working because there are several different user experiences that can all be thought of as snapping. Do you mean you want the windows to jump to each other or do you just want miniwindows to move in 5 (or some other number) pixel increments?

**Comment by fiendish** [comment]
miniwindow border snapping will be in the next release. no grid though.
